### PR TITLE
[ADD] Add module account_invoice_recompute_currency

### PR DESCRIPTION
**Pull request by novawish**
Allow users to update and convert new currency of invoice based on currency exchange rate

### CODE REVIEW

File: account_invoice_recompute_currency/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+{
+    'name': 'Account Invoice - Recompute Currency',
+    'version': '10.0.1.0.0',
+    'category': 'Accounting & Finance',
+    'summary': 'Allows to change currency of Invoice by wizard',
+    'author': 'Komit Consulting, Odoo Community Association (OCA)',
+    'website': 'https://komit-consulting.com',
+    'license': 'AGPL-3',
+    'depends': [
+        'base',
```
Comment (jcdrubay) [inline account_invoice_recompute_currency/__manifest__.py:13]: If we depend on account, we don't need to also depend on base.

File: account_invoice_recompute_currency/wizard/wizard_recompute_invoice_currency.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    </record>
+
+    <record id="action_wizard_recompute_currency" model="ir.actions.act_window">
+        <field name="name">Recompute currency of the Invoice</field>
+        <field name="res_model">wizard.recompute.invoice.currency</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">form</field>
+        <field name="target">new</field>
+    </record>
+</odoo>
```
Comment (hieulucky111) [inline account_invoice_recompute_currency/wizard/wizard_recompute_invoice_currency.xml:31]: Missing empty line.

**Review by hieulucky111 — APPROVED** [review]
LGTM Overall, just had minor adjustment.

**Review by huyly0909 — APPROVED** [review]
Everything seems right.

**Review by jcdrubay — APPROVED** [review]
Test on runbot, it works well.
Only a minor comment.

**Comment by jcdrubay** [comment]
@OCA/core-maintainers : could you please help to review?

**Comment by pedrobaeza** [comment]
@jcdrubay I'm seeing several PRs with "batch approvals" from your team. Can they be trusted as full reviews?

**Comment by jcdrubay** [comment]
Hi @pedrobaeza we did a code review session in team this morning.

In this specific PR, I actually did the test on runbot and code review.

In some others PRs, it was a simple notifications that a PR was opened for a while with some approvals, only waiting for action from maintainers.

**Comment by pedrobaeza** [comment]
OK, I just want it to be sure :)

Seeing the README, I think the module name should be `account_invoice_change_currency` and it should be at account-invoicing, don't you think?

**Comment by jcdrubay** [comment]
I fully agree with both.
All your hours of work on OCA have sharpen you code reviewer eyes to perfection ;)
We still have a lot to improve. Thanks for your patience and support.

**Comment by pedrobaeza** [comment]
Thanks to you for contributing to the community!

**Comment by novawish** [comment]
@pedrobaeza Thanks very much for your comments, I created a PR to repo `account-invoicing`: https://github.com/OCA/account-invoicing/pull/393

**Comment by pedrobaeza** [comment]
Closing this one then.

**Pull request closed without merging**